engine/src/rocksdb_engine.rs

```diff
@@ -279,6 +279,11 @@ impl RocksSnapshot {
     }
 
     /// Write snapshot data
+    /// # Errors
+    /// Return `io::Error` when write failed with io error.
+    /// # Panics
```

Comment:
Shall we not panic when got a invalid snapshot file size? I think report an error is better.

Reply:
Fixed